cmd/list/machinepool/machinepool.go

```diff
@@ -33,6 +33,12 @@ func listMachinePools(r *rosa.Runtime, clusterKey string, cluster *cmv1.Cluster)
 				MaxReplicas(cluster.Nodes().AutoscaleCompute().MaxReplicas()),
 		)
 	}
+
+	// In case of AWS clusters we can query the subnbets
+	if cluster.AWS() != nil && len(cluster.AWS().SubnetIDs()) > 0 {
```

Comment:
I'm okay considering that's how we are dealing with default subnet already. I wonder why we don't return it back within the machine pool endpoint

Reply:
I'm afraid it's by design; in hive clusters, the "Default" machinepool is built in rosa from the cluster itself.
The bug I try to fix is to add the subnets that are missing in the list.